sklearn/calibration.py

```diff
@@ -137,6 +137,14 @@ class CalibratedClassifierCV(BaseEstimator, ClassifierMixin,
     >>> calibrated_clf.predict_proba([[-0.5, 0.5]])
     array([[0.936..., 0.063...]])
 
+    label_binarizer_ : LabelBinarizer object
+        Object used to transform labels to a binary presence matrix and
+        vice-versa.
+
+    y_type_ : str
+        Represents the type of the target data as evaluated by
+        :attr:`label_binarizer_`.
+
```

Comment:
Thanks @connorbrinton! `label_binarizer_` and `y_type_` are attributes. Their description should be moved before the example. This will fix the sphinx warnings too.